data_steward/constants/validation/participants/validate.py

```diff
@@ -232,3 +430,57 @@
+    upd.email = `{{project_id}}.{{drc_dataset_id}}.CompareEmail`(ps.email, ehr_email.email),
+    upd.phone_number = `{{project_id}}.{{drc_dataset_id}}.ComparePhoneNumber`(ps.phone_number, ehr_phone.phone_number),
+    upd.birth_date = `{{project_id}}.{{drc_dataset_id}}.CompareDateOfBirth`(ps.date_of_birth, ehr_dob.date_of_birth),
+    upd.sex = `{{project_id}}.{{drc_dataset_id}}.CompareSexAtBirth`(ps.sex, ehr_sex.sex),
+    upd.algorithm = 'yes'
+FROM `{{project_id}}.{{drc_dataset_id}}.{{ps_api_table_id}}` ps
+LEFT JOIN ( WITH dedup AS ( SELECT *, row_number() OVER(PARTITION BY person_id) AS row_num
+                            FROM `{{project_id}}.{{ehr_ops_dataset_id}}.{{hpo_pii_name_table_id}}` )
+            SELECT * EXCEPT (row_num) FROM dedup WHERE row_num = 1 ) ehr_name
+    ON ehr_name.person_id = ps.person_id
+LEFT JOIN ( SELECT person_id, address_1, address_2, city, state, zip
+            FROM ( WITH dedup AS ( SELECT *, row_number() OVER(PARTITION BY person_id) AS row_num
+                                    FROM `{{project_id}}.{{ehr_ops_dataset_id}}.{{hpo_pii_address_table_id}}` )
+                    SELECT * EXCEPT (row_num) FROM dedup WHERE row_num = 1 )
+            LEFT JOIN ( WITH dedup AS ( SELECT *, row_number() OVER(PARTITION BY location_id) AS row_num
+                                        FROM `{{project_id}}.{{ehr_ops_dataset_id}}.{{hpo_location_table_id}}` )
+                        SELECT * EXCEPT (row_num) FROM dedup WHERE row_num = 1 )
+                USING (location_id) ) ehr_address
+    ON ehr_address.person_id = ps.person_id
+LEFT JOIN ( WITH dedup AS ( SELECT *, row_number() OVER(PARTITION BY person_id) AS row_num
+                            FROM `{{project_id}}.{{ehr_ops_dataset_id}}.{{hpo_pii_email_table_id}}` )
+            SELECT * EXCEPT (row_num) FROM dedup WHERE row_num = 1 ) ehr_email
+    ON ehr_email.person_id = ps.person_id
+LEFT JOIN ( WITH dedup AS ( SELECT *, row_number() OVER(PARTITION BY person_id) AS row_num
+                            FROM `{{project_id}}.{{ehr_ops_dataset_id}}.{{hpo_pii_phone_number_table_id}}` )
+            SELECT * EXCEPT (row_num) FROM dedup WHERE row_num = 1 ) ehr_phone
+    ON ehr_phone.person_id = ps.person_id
+LEFT JOIN ( SELECT person_id, DATE(birth_datetime) AS date_of_birth
+           FROM `{{project_id}}.{{ehr_ops_dataset_id}}.{{hpo_person_table_id}}` ) AS ehr_dob
+    ON ehr_dob.person_id = ps.person_id
+LEFT JOIN ( SELECT person_id, cc.concept_name as sex
+            FROM `{{project_id}}.{{ehr_ops_dataset_id}}.{{hpo_person_table_id}}`
+            JOIN `{{project_id}}.{{ehr_ops_dataset_id}}.concept` cc
+                ON gender_concept_id = concept_id ) AS ehr_sex
+    ON ehr_sex.person_id = ps.person_id
+WHERE upd.person_id = ps.person_id
+    AND upd._PARTITIONTIME = ps._PARTITIONTIME
+""")
+
+SUMMARY_QUERY = JINJA_ENV.from_string("""
+SELECT COUNT(*) AS row_count
+FROM `{{project_id}}.{{drc_dataset_id}}.{{hpo_id}}`
```

Comment:
I don't think the variables in this query match with the expectations of the `get_participant_validation_summary_query` function. `drc_dataset_id` here would need to be changed to `dataset_id` and `hpo_id` would need to be changed to `id_match_table`. Am I interpreting this correctly?

Reply:
You are right, nice catch. I wasn't using this anywhere since it is too generic, but this needs to be changed in case we do plan to use it.